apps/ledger-live-mobile/src/families/ethereum/EditTransaction/MethodSelection.tsx

```diff
@@ -0,0 +1,278 @@
+import invariant from "invariant";
+import { Trans, useTranslation } from "react-i18next";
+import BigNumber from "bignumber.js";
+import { Linking } from "react-native";
+import { Flex, SelectableList } from "@ledgerhq/native-ui";
+import { fromTransactionRaw } from "@ledgerhq/live-common/transaction/index";
+import {
+  Transaction,
+  TransactionRaw,
+} from "@ledgerhq/live-common/families/ethereum/types";
+import useBridgeTransaction from "@ledgerhq/live-common/bridge/useBridgeTransaction";
+import { Account } from "@ledgerhq/types-live";
+import { EIP1559ShouldBeUsed } from "@ledgerhq/live-common/families/ethereum/transaction";
+import {
+  getAccountCurrency,
+  getMainAccount,
+} from "@ledgerhq/live-common/account/index";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import { isOldestEditableOperation } from "@ledgerhq/live-common/operation";
+import { apiForCurrency } from "@ledgerhq/live-common/families/ethereum/api/index";
+import { TransactionHasBeenValidatedError } from "@ledgerhq/errors";
+import { getEnv } from "@ledgerhq/live-env";
+import { log } from "@ledgerhq/logs";
+
+import { ScreenName } from "../../../const";
+import { TrackScreen } from "../../../analytics";
+import { EditTransactionParamList } from "../editTransactionNavigatorParamsList";
+import { StackNavigatorProps } from "../../../components/RootNavigator/types/helpers";
+import LText from "../../../components/LText";
+import { urls } from "../../../config/urls";
+
+type Props = StackNavigatorProps<
+  EditTransactionParamList,
+  ScreenName.EditTransactionMethodSelection
+>;
+
+function MethodSelectionComponent({ navigation, route }: Props) {
+  const { operation, account, parentAccount } = route.params;
+
+  invariant(
+    operation.transactionRaw,
+    "operation.transactionRaw not found. Could not edit the transaction",
+  );
+
+  const mainAccount = getMainAccount(account, parentAccount);
+
+  const transactionToEdit = fromTransactionRaw(
+    operation.transactionRaw as TransactionRaw,
+  ) as Transaction;
+
+  const { transaction, setTransaction, status } =
+    useBridgeTransaction<Transaction>(() => {
+      return {
+        account,
+        parentAccount,
+        transaction: transactionToEdit,
+      };
+    });
+
+  invariant(
+    transaction,
+    "[useBridgeTransaction - MethodSelection] could not found transaction from bridge.",
+  );
+
+  const feePerGas = new BigNumber(
+    EIP1559ShouldBeUsed(mainAccount.currency)
+      ? transactionToEdit.maxFeePerGas!
+      : transactionToEdit.gasPrice!,
+  );
+
+  const feeValue = new BigNumber(
+    transactionToEdit.userGasLimit || transactionToEdit.estimatedGasLimit || 0,
+  )
+    .times(feePerGas)
+    .div(new BigNumber(10).pow(mainAccount.unit.magnitude));
+
+  const haveFundToCancel = mainAccount.balance.gt(feeValue.times(1.3));
+  const haveFundToSpeedup = mainAccount.balance.gt(
+    feeValue
+      .times(1.1)
```

Comment:
Same thing here with:
- `EDIT_TX_EIP1559_MAXPRIORITYFEE_GAP_SPEEDUP_FACTOR` on the `maxPriorityFee` (and that "gap" should be added on the `maxFeePerGas`)
- or `EDIT_TX_NON_EIP1559_GASPRICE_GAP_SPEEDUP_FACTOR` on the `gasPrice` 

depending on EIP1559 or not